pkgs/development/python-modules/pytest-tap/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, pytest
+, tappy
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-tap";
+  version = "3.3";
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "5f0919a147cf0396b2f10d64d365a0bf8062e06543e93c675c9d37f5605e983c";
+  };
+
+  propagatedBuildInputs = [ pytest tappy ];
```

Comment:
`pytest` should not be propagated for `pytest-*` module as it prevents overriding. Move it to `buildInputs`.

Reply:
I see... That makes sense.